frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx

```diff
@@ -63,18 +60,33 @@ export const NetworkPolicyPeerSelectors: React.FC<PeerSelectorProps> = (props) =
         />
       </div>
       <p>
-        <Trans ns="console-app" i18nKey="netpolicy_peers_preview_affected_pods">
-          Show a preview of the{' '}
-          <Button
-            data-test={`show-affected-pods-${props.direction}`}
-            ref={podsPreviewPopoverRef}
-            variant="link"
-            isInline
-          >
-            affected pods
-          </Button>{' '}
-          that this <RuleName /> will apply to
-        </Trans>
+        {props.direction === 'ingress' ? (
+          <Trans ns="console-app">
+            Show a preview of the{' '}
+            <Button
+              data-test={`show-affected-pods-${props.direction}`}
+              ref={podsPreviewPopoverRef}
+              variant="link"
+              isInline
+            >
+              affected pods
+            </Button>{' '}
+            that this ingress rule will apply to.
+          </Trans>
+        ) : (
+          <Trans ns="console-app">
+            Show a preview of the{' '}
+            <Button
+              data-test={`show-affected-pods-${props.direction}`}
```

Comment:
minor comment: maybe you can replace this line by:
```suggestion
              data-test={`show-affected-pods-egress`}
```

Reply:
I would drop the `s here and move to a string. 

```suggestion
              data-test="show-affected-pods-egress"
```